dali/operators/generic/slice/slice.cc

```diff
@@ -29,14 +29,14 @@ coordinates and ``WH`` order for the slice arguments.)code")
     .NumInput(3)
     .NumOutput(1)
```

Comment:
Since I touched the docs, I updated all the docs according to our documentation PR first, to avoid conflicts later